modules/ob-about-build-strategies.adoc

```diff
@@ -0,0 +1,40 @@
+// Ths module is included in the following assembly:
+//
+// * builds/overview-openshift-builds.adoc
+
+:_content-type: CONCEPT
+[id="about-build-strategies_{context}"]
+= About build and cluster build strategies
+
+The `BuildStrategy` and `ClusterBuildStrategy` resources define a series of steps to assemble an application. You can use the `BuildStrategy` resources within a namespace and the `ClusterBuildStrategy` resources within a cluster.
+
+The specification of a `BuildStrategy` or `ClusterBuildStrategy` resource consists of a `steps` object. The following example shows the specification of the `Kaniko` cluster build strategy:
+[source,yaml]
```

Comment:
Since below it's `name: kaniko`, should the reference to `Kaniko` here be lowercase?

```suggestion
The specification of a `BuildStrategy` or `ClusterBuildStrategy` resource consists of a `steps` object. The following example shows the specification of the `kaniko` cluster build strategy:

[source,yaml]
```

^ Also suggest adding a blank line before the `[source,yaml]` here.